spec/tapioca/compilers/_symbol_table_compiler_spec.rb

```diff
@@ -1703,5 +1703,120 @@ def foo(*_, &_); end
         RUBY
       )
     end
+
+    it("compiles signatures and structs in source files") do
+      expect(
+        compile(template(<<~RUBY))
+          class Foo
+            extend(T::Sig)
+
```

Comment:
Might be good to also test signatures on attributes to show the current behavior?

Reply:
Do you mean `attr_reader`/`attr_writer` kind of thing? If so, I can add those.

Reply:
^ Yes

Reply:
Added